tasks/kernel_matrix_testing/download.py

```diff
@@ -134,7 +138,11 @@ def download_rootfs(ctx: Context, rootfs_dir: PathOrStr, vmconfig_template_name:
         raise Exit("Failed to set permissions 0766 to rootfs")
 
 
-def update_rootfs(ctx: Context, rootfs_dir: PathOrStr, vmconfig_template: str):
-    download_rootfs(ctx, rootfs_dir, vmconfig_template)
+def update_rootfs(ctx: Context, rootfs_dir: PathOrStr, vmconfig_template: str, all_archs: bool = False):
+    if all_archs:
```

Comment:
Having to download VM images for both architectures for this generation approach to work can also be an issue. People have complained previously that KMT's disk usage is too high. Doubling this is not great.

Reply:
In theory it should only be us doing it. Although now that you said it I checked at the disk usage for the images... 125GB :D